### PR TITLE
New package: WGlunz.pstoedit version 4.0.0.0

### PR DESCRIPTION
**Pull request by woglu**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.5 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.5.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

---

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/118481)

### CODE REVIEW

**Comment by woglu** [comment]
> 
@microsoft-github-policy-service agree

**Comment by woglu** [comment]
> @woglu please read the following Contributor License Agreement(CLA). If you agree with the CLA, please reply with the following information.
> 
> ```
> @microsoft-github-policy-service agree [company="{your company}"]
> ```
> 
> > Options:
> > 
> > * (default - no company specified) I have sole ownership of intellectual property rights to my Submissions and I am not making Submissions in the course of work for my employer.
> > 
> > ```
> > @microsoft-github-policy-service agree
> > ```
> > 
> > 
> >     
> >       
> >     
> > 
> >       
> >     
> > 
> >     
> >   
> > 
> > * (when company given) I am making Submissions in the course of work for my employer (or my employer has intellectual property rights in my Submissions by contract or applicable law). I have permission from my employer to make Submissions and enter into this Agreement on behalf of my employer. By signing below, the defined term “You” includes me and my employer.
> > 
> > ```
> > @microsoft-github-policy-service agree company="Microsoft"
> > ```
> 
> Contributor License Agreement

@microsoft-github-policy-service agree

**Comment by stephengillie** [comment]
Hi @woglu,

It appears that this package has a dependency on `VCRedist`:
image

File: manifests/w/WGlunz/pstoedit/4.0.0.0/WGlunz.pstoedit.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.5.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.5.0.schema.json
+
+PackageIdentifier: WGlunz.pstoedit
+PackageVersion: 4.0.0.0
+InstallerType: inno
+Installers:
```
Comment (stephengillie) [inline manifests/w/WGlunz/pstoedit/4.0.0.0/WGlunz.pstoedit.installer.yaml:7]:
```suggestion
Dependencies:
  PackageDependencies:
    - PackageIdentifier: Microsoft.VCRedist.2015+.x64

Installers:
```

File: manifests/w/WGlunz/pstoedit/4.0.0.0/WGlunz.pstoedit.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.5.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.5.0.schema.json
+
+PackageIdentifier: WGlunz.pstoedit
+PackageVersion: 4.0.0.0
+PackageLocale: en-US
+Publisher: H&W Glunz
+PackageName: pstoedit
```
Comment (stephengillie) [inline manifests/w/WGlunz/pstoedit/4.0.0.0/WGlunz.pstoedit.locale.en-US.yaml:8]:
```suggestion
PackageName: pstoedit
PackageUrl: https://sourceforge.net/projects/pstoedit
```

**Comment by woglu** [comment]
> Hi @woglu,
> 
> It appears that this package has a dependency on `VCRedist`: image

That is true and the changes you proposed are perfect. Thanks.
Wolfgang

**Review by stephengillie — APPROVED** [review]

**Pull request merged**